doc/source/dev/roadmap-detailed.rst

```diff
@@ -256,9 +256,8 @@ Overall this module is in good shape. Two good global optimizers were added in
 1.2.0; large-scale optimizers is still a gap that could be filled.  Other
 things that are needed:
 
-- Many ideas for additional functionality (e.g. integer constraints, sparse
-  matrix support, performance improvements) in ``linprog``, see
-  `gh-9269 <https://github.com/scipy/scipy/issues/9269>`__.
+- Many ideas for additional functionality (e.g. integer constraints) in
+  ``linprog``, see `gh-9269 <https://github.com/scipy/scipy/issues/9269>`__.
```

Comment:
With the addition of HIGHS, `linprog` is now reasonably performant and has good support for sparse inputs.